sdk/communication/AzureCommunication/Tests/ObjCCommunicationTokenCredentialAsyncTests.m

```diff
@@ -106,10 +115,18 @@ - (void)xtest_ObjCRefreshTokenProactivelyTokenExpiringInNineMin {
         XCTAssertEqual(accessToken.token, weakSelf.sampleToken);
         XCTAssertEqual(weakSelf.fetchTokenCallCount, 1);
         
-        [expectation fulfill];
+        isComplete = YES;
```

Comment:
nit: should this TRUE or FALSE instead?

Reply:
In ObjC it is `YES` `NO`. I think you can use `TRUE` `FALSE` but they are just aliases for YES and NO